Wrapping/Generators/Python/CMakeLists.txt

```diff
@@ -151,9 +151,9 @@ if(NOT EXTERNAL_WRAP_ITK_PROJECT)
     itkHelpers
     itk/__init__
     itk/itkInitHelpers
+    itk_generate_pyi # A utility function used to generate the intreface file
```

Comment:
These are a list of files installed and needed for the runtime -- this build-time script does not need to be installed.

Reply:
I think we would want this installed as well.   I think that the implied content of the itk package (from __init__.py) dynamically changes based on which modules (or remote modules) are pip installed.  I was thinking that this would need to also be installed so that a savvy user could update the __iniit__.pyi file if additional remote modules are installed.